packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -73,20 +76,27 @@ function transformMetricKind(kind: OTMetricKind): MetricKind {
   switch (kind) {
     case OTMetricKind.COUNTER:
     case OTMetricKind.OBSERVABLE_COUNTER:
+    case OTMetricKind.HISTOGRAM:
       return MetricKind.CUMULATIVE;
```

Comment:
Histogram when converted to a GCM Distribution is apparently best represented by MetricKind.CUMULATIVE, as MetricKind.DELTA is not allowed for Custom Metrics.